utils.py

```diff
@@ -398,20 +405,30 @@ def get_time_in_millisecs(datetime_obj):
         datetime_obj: datetime. An object of type datetime.datetime.
 
     Returns:
-        float. This returns the time in the millisecond since the Epoch.
+        float. This returns the time in the milliseconds since the Epoch.
     """
     seconds = time.mktime(datetime_obj.utctimetuple()) * 1000
     return seconds + python_utils.divide(datetime_obj.microsecond, 1000.0)
 
 
 def get_current_time_in_millisecs():
-    """Returns time in milliseconds since the Epoch."""
+    """Returns time in milliseconds since the Epoch.
+
+    Returns:
+        float. This returns the time in the milliseconds since the Epoch.
+    """
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
 def get_human_readable_time_string(time_msec):
     """Given a time in milliseconds since the epoch, get a human-readable
     time string for the admin dashboard.
+
+    Args:
+        time_msec: float. Time in milliseconds since the Epoch.
+
+    Returns:
+        str. Returns a string representing the time.
```

Comment:
Ditto here. No need to have 'Returns a', since you need to be  explaining the returned variable itself here.